index.html

```diff
@@ -1197,63 +1197,74 @@ <h2>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
         <p>Depending on the specific hardware and platform, user agents will likely only receive one set of values for the transducer orientation relative to the screen plane - either <code>tiltX</code> / <code>tiltY</code> or <code>altitudeAngle</code> / <code>azimuthAngle</code>. The following basic code provides an initial suggested approach for converting these values.</p>
         <pre id="example_12" class="example" title="Converting between tiltX/tiltY and altitudeAngle/azimuthAngle"><code>/* Converting between tiltX/tiltY and altitudeAngle/azimuthAngle */
 
-function spherical2tilt(altitudeAngle, azimuthAngle) {
-  var tiltX = Math.atan(Math.cos(azimuthAngle) / Math.tan(altitudeAngle)) * 180 / Math.PI;
-  var tiltY = Math.atan(Math.sin(azimuthAngle) / Math.tan(altitudeAngle)) * 180 / Math.PI;
-  return [tiltX, tiltY]
+function spherical2tilt(altitudeAngle, azimuthAngle){
```

Comment:
Reviewing I realized I did not originally consider edge cases for `spherical2tilt`. The concern is when `altitudeAngle` is zero because `Math.tan(altitudeAngle)` will also be zero and the formula won't work (as we divide by `Math.tan(altitudeAngle)`). This case describes the pen flat on the surface of `X-Y` plane. When pen is on the `X` axis, `tiltY` doesn't really exist so I set it to zero. When pen is on `Y` axis, `tiltX` doesn't really exist and I set it to zero. The reason the tilt angles do not exist is because the pen and the axis form one line, and we need at least two lines to have a plane in order to calculate an angle.  

Javascript code here: https://gist.github.com/liviutinta/0d040270ecf3390972421cb2164fbf37